odoo/addons/test_read_group/models.py

```diff
@@ -65,3 +65,31 @@ class OrderLine(models.Model):
 
     order_id = fields.Many2one('test_read_group.order', ondelete='cascade')
     value = fields.Integer()
+
+
+class User(models.Model):
+    _name = 'test_read_group.res.users'
```

Comment:
Smaller model names will lead to simpler aliases :wink: 
`test_read_group.res.users` → `test_read_group.user`
`test_read_group.project.task` → `test_read_group.task`